Sample/App/Examples/OnDeviceEvaluations/SdkDemoTypes.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
```

Comment:
note: manually written by the implementor, but could be code generated by Statsig